kudos-services/src/main/java/org/exoplatform/kudos/listener/KudosActivityListener.java

```diff
@@ -0,0 +1,71 @@
+package org.exoplatform.kudos.listener;
+
+import lombok.SneakyThrows;
+import org.exoplatform.commons.persistence.impl.GenericDAOJPAImpl;
+import org.exoplatform.kudos.dao.KudosDAO;
+import org.exoplatform.kudos.entity.KudosEntity;
+import org.exoplatform.kudos.model.Kudos;
+import org.exoplatform.kudos.service.KudosService;
+import org.exoplatform.kudos.service.KudosStorage;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.social.core.activity.ActivityLifeCycleEvent;
+import org.exoplatform.social.core.activity.ActivityListenerPlugin;
+import org.exoplatform.social.core.activity.model.ExoSocialActivity;
+
+import java.util.Map;
+
+public class KudosActivityListener extends ActivityListenerPlugin {
+    private static final Log LOG = ExoLogger.getLogger(KudosActivityListener.class);
+    private static String activityType ="exokudos:activity";
```

Comment:
remove this constant and use org.exoplatform.kudos.service.utils.Utils#KUDOS_ACTIVITY_COMMENT_TYPE